### PR TITLE
Notification screen design updates

### PR DESCRIPTION
**Pull request by manojava-gk**

### CODE REVIEW

File: cx-portal/src/components/pages/NotificationCenter/DeleteNotificationConfirmOverlay.tsx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+          variant="text"
+          onClick={(e) => handleClose(e)}
+          size="medium"
+          sx={{
+            fontSize: '14px',
+            marginRight: '15px',
+            borderRadius: '8px',
+          }}
+        >
+          CANCEL
```
Comment (oyo) [inline cx-portal/src/components/pages/NotificationCenter/DeleteNotificationConfirmOverlay.tsx:79]: i18n?

File: cx-portal/src/components/pages/NotificationCenter/DeleteNotificationConfirmOverlay.tsx [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+          color="error"
+          variant="contained"
+          onClick={(e) => handleClose(e)}
+          size="large"
+          sx={{
+            borderRadius: '8px',
+            fontSize: '14px',
+          }}
+        >
+          DELETE ALWAYS
```
Comment (oyo) [inline cx-portal/src/components/pages/NotificationCenter/DeleteNotificationConfirmOverlay.tsx:91]: i18n?

**Review by oyo — APPROVED** [review]
some texts not translated - guess that will come later?

File: cx-portal/src/components/pages/NotificationCenter/DeleteNotificationConfirmOverlay.tsx (hunk 1 above)
Comment (manojava-gk) [reply cx-portal/src/components/pages/NotificationCenter/DeleteNotificationConfirmOverlay.tsx:79]: Oops. Will update the PR.

**Comment by jjeroch** [comment]
@manojava-gk please check the sorting element => ideally a sharedUiComponent element?

**Comment by manojava-gk** [comment]
> 

Okay sure. Will create it

**Review by lavanya-bmw — APPROVED** [review]
lgtm

**Pull request merged**